pkgs/development/libraries/libucl/default.nix

```diff
@@ -0,0 +1,31 @@
+{ stdenv, fetchFromGitHub, pkgconfig, automake, autoconf, libtool }:
+
+let
+  pname = "libucl";
+  version = "0.8.1";
+in
+
+stdenv.mkDerivation {
+  inherit pname version;
+
+  nativeBuildInputs = [ pkgconfig autoconf automake libtool ];
```

Comment:
Interesting! I started with this, but the build did not go through. I think that was because the configuration script wasn't called, the Makefile not generated, and in the end, the build phase did nothing. I'll try again.